rero_ils/modules/patron_transactions/jsonschemas/patron_transactions/patron_transaction-v0.0.1.json

```diff
@@ -114,27 +114,31 @@
         }
       }
     },
-    "loan": {
-      "title": "Loan",
+    "notification": {
+      "title": "Notification",
       "type": "object",
       "properties": {
         "$ref": {
-          "title": "Loan URI",
+          "title": "Notification URI",
           "type": "string",
-          "pattern": "^https://ils.rero.ch/api/loans/.*?$"
+          "pattern": "^https://ils.rero.ch/api/notifications/.*?$"
         }
       }
     },
-    "notification": {
-      "title": "Notification",
+    "organisation": {
+      "title": "Organisation",
       "type": "object",
       "properties": {
         "$ref": {
-          "title": "Notification URI",
+          "title": "Organisation URI",
           "type": "string",
-          "pattern": "^https://ils.rero.ch/api/notifications/.*?$"
+          "pattern": "^https://ils.rero.ch/api/organisations/.*?$"
         }
       }
+    },
+    "total_amount": {
+      "type": "number",
+      "title": "Total calculated amount of the patron transaction"
```

Comment:
much pretty if you add `"minimum": 0` ... just to be sure

Reply:
fixed